src/Sarif.Multitool.FunctionalTests/Update-Expected.ps1

```diff
@@ -1,28 +1,34 @@
 ﻿<#
 .SYNOPSIS
-Update the expected output files for the SarifCli functional tests.
+Update the expected output files for the Sarif.Multitool functional tests.
 
 .DESCRIPTION
-This script runs "SarifCli validate" on each input log file in the SarifCli
+This script runs "Sarif.Multitool validate" on each input log file in the SarifCli
 functional tests TestData directory. For each file foo.sarif, it produces
 an output file foo_Expected.sarif, overwriting any existing file of that
 name in the TestData directory.
 
-.PARAMETERS
-    Configuration
-        The build configuration. Default: Debug
+.PARAMETER Configuration
+    The configuration of the build that produced Sarif.Multitool.exe.
+    Default: Debug
+
+.PARAMETER RuleName
+    The name of the rule whose files are to be regenerated.
+    Default: $null, meaning "all rules"
 #>
 
-param([string] $Configuration = "Debug")
+param(
+    [string] $Configuration = "Debug",
+    [string] $RuleName = $null
+)
 
 $ErrorActionPreference = "Stop"
 
-$SarifExtension = ".sarif"
+Import-Module -Force ..\..\scripts\ScriptUtilities.psm1
```

Comment:
`$SarifExtension` and `$JsonSchemaPath` are now exported from `ScriptUtilities.psm1`.